Makefile

```diff
@@ -76,15 +76,26 @@ $(PROTOC):
 	unzip  -d $(PROTOC_DIR) $(PROTOC_DIR)/$(PROTOC_ZIP_NAME)
 	rm $(PROTOC_DIR)/$(PROTOC_ZIP_NAME)
 
+GOTEST_INTEGRATION = $(GOTEST) -tags=integration
+
 # Integration test for sdk client
 .PHONY: test-sdk
 test-sdk: clean-sdk build-sdk
-	DUMMY_BIN=$(DUMMY_BIN) LOOKOUT_BIN=$(LOOKOUT_BIN) $(GOCMD) run cmd/sdk-test/main.go
+	DUMMY_BIN=$(PWD)/$(DUMMY_BIN) \
+	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_BIN) \
+	$(GOTEST_INTEGRATION) github.com/src-d/lookout/cmd/sdk-test
+
+# Same as test-sdk, but skipping tests that require a bblfshd server
+.PHONY: test-sdk-macos
```

Comment:
May be `test-sdk-short` would make it a bit more clear? As one may want to run short version on any os

Reply:
:+1: done in 5b4e0b4.